addons/website/static/src/snippets/s_image_gallery/000.js

```diff
@@ -139,7 +136,9 @@ const GallerySliderWidget = publicWidget.Widget.extend({
             page += ($(this).hasClass('o_indicators_left') ? -1 : 1);
             page = Math.max(0, Math.min(nbPages - 1, page)); // should not be necessary
             self.$carousel.carousel(page * realNbPerPage);
-            hide();
+            if (!self.editableMode) {
+                hide();
+            }
```

Comment:
We don't do it in the editor because there is 1 traceback. I don't know why it seems related to summernote.
And since it's more a visual need that does not change enormously, I just deactivate it in the editor. I add a comment